manager/integration/tests/test_migration.py

```diff
@@ -153,7 +157,8 @@ def test_migration_with_unscheduled_replica(clients, volume_name):  # NOQA
                                   backingImage="",
                                   accessMode="rwx", migratable=True)
     volume = common.wait_for_volume_detached(client, volume_name)
-    volume.attach(hostId=local_node)
+    attachment_id = common.generate_attachment_ticket_id()
+    volume.attach(hostId=local_node, attachmentID=attachment_id)
```

Comment:
Do we miss `attachmentType`?

Reply:
By default, if no type is set, it will be `longhorn-api` type with attachmentID `""`